src/services/edusharing/index.js

```diff
@@ -25,20 +32,28 @@ class MerlinToken {
 
 module.exports = (app) => {
 	const eduSharingRoute = '/edu-sharing';
-	const merlinRoute = `${eduSharingRoute}/merlinToken`;
-	const docRoute = `${eduSharingRoute}/api`;
+	const eduSharingPlayerRoute = '/edu-sharing/player';
+	const merlinRoute = '/edu-sharing/merlinToken';
+	const docRoute = '/edu-sharing/api';
 
-	app.use(docRoute, staticContent(path.join(__dirname, '/docs/openapi.yaml')));
+	app.use(eduSharingRoute, new EduSharing(), (req, res) => {
+		res.send(res.data);
+	});
+	const eduSharingService = app.service(eduSharingRoute);
+	eduSharingService.hooks(hooks);
+
+	app.use(eduSharingPlayerRoute, new EduSharingPlayer(), (req, res) => {
+	    res.send(res.data);
```

Comment:
Why you add a service to feathersJS with hooks and handling and then you add a express middelware add the end for res.send? Same in line 40.